aea/aea_builder.py

```diff
@@ -445,6 +446,33 @@ def set_decision_maker_handler(
 
         return self
 
+    def set_error_handler(
+        self, error_handler_dotted_path: str, file_path: Path
+    ) -> "AEABuilder":
+        """
+        Set error handler class.
+
+        :param error_handler_dotted_path: the dotted path to the error handler
+        :param file_path: the file path to the file which contains the error handler
+
+        :return: self
+        """
+        dotted_path, class_name = error_handler_dotted_path.split(":")
```

Comment:
I just found a bug there in the decision maker handler code too! it should be `::`!

Reply:
Please fix here and in dm case

Reply:
(line 432)

Reply:
Why `::` and not `:`?

Reply:
OK, I can see scaffold uses `::`. However, not sure about this choice.

Reply:
pytest also uses ::

Reply:
I don't have anything about `::`. However, it is an arbitrary choice.
The argument "pytest uses it" can be answered by saying that:
- `pydoc.locate` doesn't even use colons, just dots: https://github.com/python/cpython/blob/master/Lib/pydoc.py#L1708-L1725
- `gym` registry uses `:`
- We do use `:` in registry entrypoint: https://github.com/fetchai/agents-aea/blob/master/aea/crypto/registries/base.py#L68